projects/train/configs/flow/cbc.yaml

```diff
@@ -0,0 +1,82 @@
+# lightning.pytorch==2.1.3
+seed_everything: 101588
+trainer:
+  accelerator: auto
+  strategy: auto
+  devices: auto
+  profiler: "simple"
+  logger:
+    - class_path: lightning.pytorch.loggers.WandbLogger
+      init_args:
+        save_dir: "/home/ethan.marx/amplfi/runs/first-lightning-run/train/"
```

Comment:
Is there a way to get this from an environment variable. Otherwise this will be a diff everytime based on who creates an MR

Reply:
Yeah good point there definitely is let me dig into it.

Reply:
We can use the link_arguments feature here with

```
compute_fn=lambda x: os.getenv(x) if x.startswith("$") else x
```
or something equivalent to attach the paths. Need to be careful of
```
    save_dir: "$AMPLFI_SAVE_DIR"
```
vs `${AMPLFI_SAVE_DIR}` but that's a minor detail.